en/getting_help/can_i_use_jquery_or_other_dom_manipulating_js.md

```diff
@@ -5,15 +5,20 @@ Yes!  At the moment Volt ships with jQuery, though this will be removed in a fut
 1. Calling JS from Volts controller's (via Opal)
 2. Using action callbacks to bind to the dom at the right times.
 
+To access the dom that volt has rendered, access has to be after the action, usually in
```

Comment:
Maybe this should explain that the action is called first, then it renders, then you can do stuff after render in _ready?